components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -410,12 +428,20 @@ class NewTabPage extends React.Component<Props, State> {
           this.setAssetUSDPrice(ticker, price)
         })
       }
-      chrome.binance.getDepositInfo(ticker, (address: string, url: string) => {
-        this.setAssetDepositInfo(ticker, address, url)
-        generateQRData(address, ticker, this.setAssetDepositQRCodeSrc)
-      })
     }
 
+    chrome.binance.getCoinNetworks((networks: Record<string, string>) => {
+      const currencies = this.getCurrencyList()
+      for (let ticker in networks) {
+        if (currencies.includes(ticker)) {
```

Comment:
We'll check for only what's in the deposit list, as there are hundreds of currencies in the network endpoints